cryptofeed/exchange/binance.py

```diff
@@ -70,9 +70,10 @@ def _reset(self):
                                      side=SELL if msg['m'] else BUY,
                                      amount=amount,
                                      price=price,
-                                     timestamp=timestamp_normalize(self.id, msg['E']))
+                                     timestamp=timestamp_normalize(self.id, msg['E']),
+                                     receipt_timestamp=timestamp)
 
-    async def _ticker(self, msg):
+    async def _ticker(self, msg: dict, timestamp: float):
```

Comment:
I'd call the variable receipt_timestamp since that's where you're passing it for consistency

Reply:
it also gets used as the timestamp if one is not available, so its still a timestamp